utils/compatibility_test.go

```diff
@@ -0,0 +1,26 @@
+package utils
+
+import (
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type rpcChainCompatibility struct {
+	RPCChainVMProtocolVersion map[string]int `json:"rpcChainVMProtocolVersion"`
+}
+
+func TestCompatibility(t *testing.T) {
+	subevmVersion := "v0.4.0"
+	expectedRPCVersion := 17
+
+	compat, err := os.ReadFile("../compatibility.json")
+	assert.NoError(t, err)
+
+	var parsedCompat rpcChainCompatibility
+	err = json.Unmarshal(compat, &parsedCompat)
+	assert.NoError(t, err)
+	assert.Equal(t, expectedRPCVersion, parsedCompat.RPCChainVMProtocolVersion[subevmVersion])
+}
```

Comment:
can we move this test to `plugin/evm/version_test.go` and use the actual version and actual imported `rpcchainvm.ProtocolVersion` from AvalancheGo, so that if we mess this up for the latest version this. test will fail?

Reply:
Added a test, for it to work tho you will need to remember to update the version in `plugins/evm/version.go`.

Reply:
That's the goal 😉